client/components/site-selector/index.jsx

```diff
@@ -283,6 +283,29 @@ const SiteSelector = React.createClass( {
 				<div className="site-selector__sites" ref="selector">
 					{ this.renderAllSites() }
 					{ this.renderSites() }
+					{ hiddenSitesCount > 0 &&
+						<span className="site-selector__hidden-sites-message">
+							{ this.translate(
+								'%(hiddenSitesCount)d more hidden site. {{a}}Change{{/a}}.{{br/}}Use search to access it.',
+								'%(hiddenSitesCount)d more hidden sites. {{a}}Change{{/a}}.{{br/}}Use search to access them.',
+								{
+									count: hiddenSitesCount,
+									args: {
+										hiddenSitesCount: hiddenSitesCount
+									},
+									components: {
+										br: <br />,
+										a: <a
+												href="https://dashboard.wordpress.com/wp-admin/index.php?page=my-blogs&show=hidden"
```

Comment:
Tabbing feels strange here, since it's two tabs indented from the previous line. I'd suggest one of:

```js
a: <a
	href=""
	{ /* ... */ }
/>
```

Or:

```js
a: (
	<a
		href=""
		{ /* ... */ }
	/>
)
```